scripts/gha/integration_testing/gameloop/gameloopUITests/gameloopUITests.swift

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright (c) 2021 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+//  gameloopUITests.swift
+//  gameloopUITests
+//
+
+import XCTest
+
+/// Constants used across the loop launcher and UI test.
+public struct Constants {
```

Comment:
Don't know much about swift but noticed the same constants are also defined in "Constants.swift". Could we reuse?